09_json/willshen8/pkg/puppy/error.go

```diff
@@ -0,0 +1,19 @@
+package puppy
+
+import "fmt"
+
+type Error struct {
```

Comment:
These could also have json tags for marshalling/unmarshalling. (depends on whether you intend to send these along with error responses when you do lab 10).

Reply:
Cool, thanks. I'll added on lab9 and will do the same for lab10